tests/scripts/run-xunit-perf.cmd

```diff
@@ -257,10 +263,12 @@ setlocal
     exit /b 1
   )
 
-  call :run_cmd py.exe "%BENCHVIEW_PATH%\upload.py" submission.json --container coreclr
-  IF %ERRORLEVEL% NEQ 0 (
-    call :print_error Uploading to BenchView failed.
-    exit /b 1
+  if defined SHOULD_UPLOAD_TO_BENCHVIEW (
+    call :run_cmd py.exe "%BENCHVIEW_PATH%\upload.py" submission.json --container coreclr
+    IF %ERRORLEVEL% NEQ 0 (
```

Comment:
Instead of these lines:
```cmd
call :run_cmd py.exe "%BENCHVIEW_PATH%\upload.py" submission.json --container coreclr
IF %ERRORLEVEL% NEQ 0 (
```
I would change it for:
```cmd
call :run_cmd py.exe "%BENCHVIEW_PATH%\upload.py" submission.json --container coreclr || (
```
Otherwise the non-delayed expansion will evaluate the %ERRORLEVEL% before entering the block and the step will always pass.